local-setup/jupyter/dataloader/crs_loader.py

```
@@ -308,40 +338,186 @@ def _bootstrap_tenant_root(self, target_root: str, source_tenant: str = "pg") ->
+                    tenant=target_root
+                )
+                created = result.get('created', 0)
+                if created > 0:
+                    print(f"   ✅ Created {created} {boundary_item['schema_code']} records")
+                else:
+                    print(f"   ⚠️  {boundary_item['schema_code']} records already exist")
+            except Exception as e:
+                print(f"   ⚠️  Error creating {boundary_item['schema_code']} data: {e}")
+
+        # Skip creating tenant.tenants in bootstrap - main create_tenant flow will create the city
+        print(f"   ✅ Skipping tenant.tenants creation (city will be created by main flow)")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)